access-protected-api/src/main/java/com/contoso/api/ApiApplication.java

```diff
@@ -0,0 +1,13 @@
+package com.contoso.api;
+
+import org.springframework.boot.SpringApplication;
+import org.springframework.boot.autoconfigure.SpringBootApplication;
+
+@SpringBootApplication
+public class ApiApplication {
+
+	public static void main(String[] args) {
+		SpringApplication.run(ApiApplication.class, args);
+	}
+
```

Comment:
shall we remove this empty line or is expected to be this in Java?

Reply:
I ran the formatter before committing this, so whatever was emitted was the standard there.

Reply:
Thanks, indeed formatting was messed up on this file, I'm not sure why the formatter didn't pick it up.  Hum.  Anyhow. Applied in https://github.com/Azure-Samples/ms-identity-docs-code-java/pull/6/commits/aafb98e1ea87dea16cd92bd7865747d4e321af22